Readded Stinson filter

rchung95 opened the pull request:
![stinson](https://cloud.githubusercontent.com/assets/10456188/19584239/f7ba1abc-9710-11e6-9286-50f8f85b1f75.png)

una commented:
This looks great :)

una commented:
For some reason, this look a bit different on my system than on yours

![screen shot 2016-11-02 at 10 04 29 am](https://cloud.githubusercontent.com/assets/1693164/19931733/f41785b6-a0e3-11e6-934f-d035c4f11864.jpg)

rchung95 commented:
I retested it and notice that. I do apologize for not noticing it sooner. I have added another commit, and ran the test. I have changed the settings and this one is a lot better.

una commented on source/scss/stinson.scss:
```diff
@@ -0,0 +1,41 @@
+//   }
+@mixin stinson($filters...) {
+  @include filter-base;
+  filter: brightness(1.25) contrast(0.8) saturate(0.95) grayscale(0.05) sepia(0.05) $filters;
```
Thanks for this PR! I did some adjustments, what do you think of using `    filter: brightness(1.15) contrast(.8) saturate(.85) sepia(.1);`

and altering the `::after` pseudo element to be:

```
background: rgba(240, 149, 128, .2);
mix-blend-mode: soft-light;
```

I think it looks a little bit closer (tiny bit lighter and warmer):
![screen shot 2016-11-02 at 11 05 37 pm](https://cloud.githubusercontent.com/assets/1693164/19954725/29e0b268-a151-11e6-92f9-2cd333ed3e20.jpg)

una commented:
I made similar comments on #251 -- there are a few PRs on this filter right now, just waiting on the first one to adjust their filter

rchung95 commented:
Added the requested changes :)
